src/inference/src/cnn_network_ngraph_impl.cpp

```diff
@@ -439,8 +439,8 @@ void CNNNetworkNGraphImpl::reshape(const std::map<std::string, ngraph::PartialSh
                 manager.register_pass<ov::pass::RemoveConcatZeroDimInput>();
                 manager.register_pass<ov::pass::RemoveMultiSubGraphOpDanglingParams>();
                 manager.register_pass<::ngraph::pass::ConvertNMS5ToLegacyMatcher>(false);
-                manager.register_pass<::ngraph::pass::ConvertMulticlassNmsToMulticlassNmsIE>();
-                manager.register_pass<::ngraph::pass::ConvertMatrixNmsToMatrixNmsIE>();
+                manager.register_pass<::ngraph::pass::ConvertMulticlassNmsToMulticlassNmsIE>(false);
+                manager.register_pass<::ngraph::pass::ConvertMatrixNmsToMatrixNmsIE>(false);
```

Comment:
Why these changes needed?

Reply:
The transformation will not insert 'convert' node to convert 'i64' to 'i32' with the parameter 'false'. If add 'convert' node, the network output name in 'reshape' will change to 'XXX_Nms_0/convert.1'. In CPU plugin when nms node followed by 'Result' node the transformation will be skipped and the network output name will be 'XXX_Nms_0', this is conflict with the previous one.
There are some ways to resolve the problem:
1, (currently used) Tell the transformation not to add 'convert' node in the caller 'reshape', followed the original [NonMaxSuppression design](https://github.com/luo-cheng2021/openvino/blob/9a5b91185679d67b83a6841603454006dc9f1d09/src/inference/src/cnn_network_ngraph_impl.cpp#L440)
2, Always add 'convert' node even if the transformation is skipped. There is a little concern: calling the transformation in CPU plugin may be removed someday.